pkg/shares/compact_shares_test.go

```diff
@@ -20,9 +20,9 @@ func TestCompactShareWriter(t *testing.T) {
 	txs := testfactory.GenerateRandomTxs(33, 200)
 	for _, tx := range txs {
 		rawTx, _ := MarshalDelimitedTx(tx)
-		w.WriteBytes(rawTx)
+		w.write(rawTx)
```

Comment:
You could just combine the two to `w.WriteTx(tx)`

Reply:
Resolved in https://github.com/celestiaorg/celestia-app/pull/1276/commits/9378d296e66a09619b1a63a8ecca63e739134cc9